src/NReco.Logging.File/NReco.Logging.File.csproj

```diff
@@ -81,23 +82,11 @@ Version 1.0.4 changes:
     <SignAssembly>false</SignAssembly>
     <AssemblyOriginatorKeyFile>NReco.Logging.File.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
-  
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
All these packages in their version 8 are compiled for `netstandard2.0`, `net6.0` and `net8.0`, so there's no need to create branches for separate frameworks.